apis/status/v1beta1/util.go

```diff
@@ -6,23 +6,34 @@ import (
 	"sync"
 )
 
+type PodOwnershipMode bool
+
+const (
+	// PodOwnershipDisabled indicates that owner references should not be put on
+	// dependent resources.
+	PodOwnershipDisabled PodOwnershipMode = false
```

Comment:
nit: This reads less intuitive as a boolean that defaults to `false`. When I read `PodOwnershipDisabled` I think if it's true it means it's disabled, not the other way around.

Reply:
This is an enum backed by a boolean. So "false" maps to "disabled" and "true" maps to "enabled".

Reply:
(I'm going to do a quick CL to make this even more clear - I may be able to delete this type entirely)